packaging/install_dependencies.sh

```diff
@@ -2,12 +2,22 @@
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 ROS_VERSION=$(rosversion -d)
-if [ "$ROS_VERSION" = "noetic" ]; then
+if [ "$ROS_VERSION" = "noetic" -o "$ROS_VERSION" = "foxy" ]; then
```

Comment:
I would rather test for not equal to "melodic". Then we don't need to change this for new ROS 2 versions